### PR TITLE
release notes for 0.4.8

### PR DESCRIPTION
**Pull request by tlambert03**
# Description
Initial release notes for 0.4.8 ... there are a few PRs and fixes that need to merge before we can cut a release ... and we discussed leaving this release as a release candidate for an extended time (~3-7 days?)  But it's over a month since 0.4.7 and I know there are some features plugin devs are waiting on.

I haven't added any "prose" to these notes yet, and could use some feedback on what people want to highlight

### CODE REVIEW

File: docs/release/release_0_4_8.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
+# napari 0.4.7
+
+We're happy to announce the release of napari 0.4.7!
```
Comment (Czaki) [inline docs/release/release_0_4_8.md]:
```suggestion
We're happy to announce the release of napari 0.4.8!
```

File: docs/release/release_0_4_8.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
+# napari 0.4.7
```
Comment (Czaki) [inline docs/release/release_0_4_8.md]:
```suggestion
# napari 0.4.8
```

**Review by Czaki — CHANGES_REQUESTED** [review]
It looks like some copy and pase bug

**Comment by jni** [comment]
I just want to reiterate that we should *not* release before addressing #2625

**Comment by tlambert03** [comment]
That was the PR I was specifically referring to in my first comment.  Not trying to rush it.  Is anyone on that?

File: docs/release/release_0_4_8.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+- Add Qt`ListModel` and `ListView` for `EventedList` (#2486)
+- New new qt layerlist (#2493)
+- Move plugin sorter (#2501)
+- Add ColorManager to Vectors (#2512)
+- PR: Enhance translation methods (#2517)
+- PR: Add typing to schema_version (#2536)
+- PR: Add initial restart implementation (#2540)
+- Add data setter for `surface` layers (#2544)
+- Extract shortcut into their own object. (#2554)
+- example of using magicgui with thread_worker (#2577)
```
Comment (jni) [inline docs/release/release_0_4_8.md]: move to documentation

File: docs/release/release_0_4_8.md [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+- Add example tying slider change to point properties change (#2582)
```
Comment (jni) [inline docs/release/release_0_4_8.md:94]: move to documentation

File: docs/release/release_0_4_8.md [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+- Range of label spinbox is more dtype-aware (#2597)
+- PR: Add generic name to unnamed dockwidgets (#2604)
+- PR: Hide ipy interactive option (#2605)
+- PR: Add option to save state separate from geometry (#2606)
+- QtLargeIntSpinbox for label controls (#2608)
+- Support varying number of dimensions during labels painting (#2609)
```
Comment (jni) [inline docs/release/release_0_4_8.md:99]: Ok I'm biased but I'd consider highlighting this. 😂

File: docs/release/release_0_4_8.md [hunk 6]
```diff
@@ -0,0 +1,157 @@
...
+- Modified quaternion2euler function to cap the arcsin's argument by +-1 (#2530)
+- single line change to track recoloring function (#2532)
+- PR: Handle Escape on Preferences dialog (#2537)
+- PR: Fix close window handling for non-modals (#2538)
+- PR: Fix trans to use new API (#2539)
+- fix set_call_order with missing plugin (#2543)
+- Update conditional to use new selection property (#2557)
+- Fix visibility toggle (and other key events) in new qt layerlist (#2561)
+- Delay importing plugins during settings registration (#2575)
+- Don't create a dask cache if it doesn't exist (#2590)
```
Comment (jni) [inline docs/release/release_0_4_8.md:128]: Potentially we can highlight this to warn users of what to do if they notice a performance drop with dask arrays.

File: docs/release/release_0_4_8.md [hunk 7]
```diff
@@ -0,0 +1,157 @@
...
+- PR: Update model and actions on menu (#2602)
+- Fix z-index of notifications (hidden by welcome window) (#2611)
+- add missing QSpinBox import in Labels layer controls (#2619)
+
+## API Changes
+
+- Remove toggle theme from drop down menu.  (#2462)
+- Add support for passing `shape_type` through `data` attribute for `Shapes` layers (#2507)
+- Deprecate gui qt (#2533)
```
Comment (jni) [inline docs/release/release_0_4_8.md]: DEFINITELY highlight this. 😊 

Also, this is not an API change yet. I think the API changes section should be reserved for code that used to work that now doesn't. So I *think* we actually don't have any API changes in this release?

Comment (tlambert03) [reply docs/release/release_0_4_8.md]: I had it in the actual "highlights" section, but then removed it as it didn't feel like a "highlight".  It's obviously going to need mentioning in the intro.  Do you want an additional bullet point in that section too?

**Comment by jni** [comment]
I've done a first pass @tlambert03! Comments inline but the biggest thing is that afaict there are no API changes, only deprecations.

Also, note that my n-dimensional painting PR does include a deprecation from `labels.n_dimensional` and `labels.events.n_dimensional` to `labels.n_edit_dimensions` and `labels.events.n_edit_dimensions`.

**Comment by tlambert03** [comment]
Thank you kindly!

File: docs/release/release_0_4_8.md (hunk 3 above)
Comment (sofroniewn) [reply docs/release/release_0_4_8.md]: done

File: docs/release/release_0_4_8.md (hunk 5 above)
Comment (sofroniewn) [reply docs/release/release_0_4_8.md:99]: done

File: docs/release/release_0_4_8.md (hunk 6 above)
Comment (sofroniewn) [reply docs/release/release_0_4_8.md:128]: Should go in API changes I think

Comment (sofroniewn) [reply docs/release/release_0_4_8.md:128]: Ah, I see it's already there, I think that's fine ...

**Comment by sofroniewn** [comment]
Ok, I've added the additional merged PRs here @jni can you take this over and write out the highlights section. Also can you go through the deprecations and any API changes and make sure everything is captured. Thanks!!

**Comment by jni** [comment]
@tlambert03 @sofroniewn this is ready for a final round of review and maybe merging.

File: docs/release/release_0_4_8.md (hunk 4 above)
Comment (sofroniewn) [reply docs/release/release_0_4_8.md:94]: @jni did you want to move this?

**Review by sofroniewn — APPROVED** [review]
All looks good to me, thanks!!!

**Comment by jni** [comment]
Nice, thanks for the update!

**Pull request merged**